packages/trader/src/Stores/Modules/MT5/Helpers/mt5-config.js

```diff
@@ -42,19 +42,19 @@ export const getMtCompanies = () => {
             synthetic: {
                 mt5_account_type: synthetic_config.account_type,
                 leverage: synthetic_config.leverage,
-                title: localize('Real Synthetic'),
+                title: localize('DMT5 Synthetic'),
```

Comment:
So are we still using the `DMT5` prefixes?

Reply:
Will remove it per above discussion.